cli/store_cmds.go

```diff
@@ -244,6 +244,57 @@ func Import(buildDataFS vfs.FileSystem, stor interface{}, opt ImportOpt) error {
 		hasIndexableData bool
 	)
 
+	importGraphData := func(graphFile string, sourceUnit *unit.SourceUnit) error {
+		var data graph.Output
+		if err := readJSONFileFS(buildDataFS, graphFile, &data); err != nil {
+			if err == errEmptyJSONFile {
+				log.Printf("Warning: the JSON file is empty for unit %s %s.", sourceUnit.Type, sourceUnit.Name)
+				return nil
+			}
+			if os.IsNotExist(err) {
+				log.Printf("Warning: no build data for unit %s %s.", sourceUnit.Type, sourceUnit.Name)
+				return nil
+			}
+			return fmt.Errorf("error reading JSON file %s for unit %s %s: %s", graphFile, sourceUnit.Type, sourceUnit.Name, err)
+		}
+		if opt.DryRun || GlobalOpt.Verbose {
+			log.Printf("# Importing graph data (%d defs, %d refs, %d docs, %d anns) for unit %s %s", len(data.Defs), len(data.Refs), len(data.Docs), len(data.Anns), sourceUnit.Type, sourceUnit.Name)
```

Comment:
This is not for debugging, but for dry runs and 'verbose' output during builds, when the `-v` argument is passed to the command.